docs/developing.md

```diff
@@ -67,17 +67,11 @@ git clone https://github.com/openshift/hive.git
 
 ## Deploying with Kubernetes In Docker (kind)
 
-[Kind](https://github.com/kubernetes-sigs/kind) can be used as a lightweight development environment for deploying and testing Hive. Currently, we support 0.5.1 version of Kind. The following instructions cover creating an insecure local registry (allowing for dramatically faster push/pull), and configuring your host OS, as well as the kind cluster to access it. This approch runs Hive in a container as you would in production, giving you the best coverage for manual testing.
+[Kind](https://github.com/kubernetes-sigs/kind) can be used as a lightweight development environment for deploying and testing Hive. Currently, we support 0.8.1 version of Kind. The following instructions cover creating an insecure local registry (allowing for dramatically faster push/pull), and configuring your host OS, as well as the kind cluster to access it. This approch runs Hive in a container as you would in production, giving you the best coverage for manual testing.
 
 This approach requires [Docker](https://docs.docker.com/install). At present we do not have kind working with podman.
 
-Deploy a local insecure registry container, and configure your host docker daemon to be able to use it:
-
-```bash
-./hack/create-insecure-registry.sh
-```
-
-Create a kind cluster named 'hive' to deploy to. You can create as many kind clusters as you need.:
+Create a local insecure registry (if one does not already exist) and then a kind cluster named 'hive' to deploy to. You can create as many kind clusters as you need and you have sufficient RAM.
```

Comment:
The part about sufficient RAM could use some grammatical work. I do not think that it is necessary to call out specifically RAM.
```suggestion
Create a local insecure registry (if one does not already exist) and then a kind cluster named 'hive' to deploy to. You can create multiple kind clusters. Each kind cluster needs a unique name, which is specified by the argument passed to the script.
```